### ISSUE TITLE
Error building MessageListenerContainer

### ISSUE THREAD

**Issue by ilyakorneckis** [Nylle/JavaFixture#18]
I get this error when trying to instantiate a spring-kafka interface `org.springframework.kafka.listener.MessageListenerContainer`

```
com.github.nylle.javafixture.SpecimenException: Unsupported type: interface java.util.Collection

	at com.github.nylle.javafixture.specimen.CollectionSpecimen.createFromInterfaceType(CollectionSpecimen.java:133)
	at com.github.nylle.javafixture.specimen.CollectionSpecimen.create(CollectionSpecimen.java:76)
	at com.github.nylle.javafixture.specimen.InterfaceSpecimen$GenericInvocationHandler.lambda$invoke$0(InterfaceSpecimen.java:62)
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1133)
	at com.github.nylle.javafixture.specimen.InterfaceSpecimen$GenericInvocationHandler.invoke(InterfaceSpecimen.java:62)
```

**Comment by Nylle** [comment]
The underlying problem is the `Collection`-interface, which was simply forgotten. 😇

**Comment by Nylle** [comment]
Fixed with release 1.2.4
